packages/components/carousel/src/carousel-item.vue

```diff
@@ -185,6 +189,7 @@ onMounted(() => {
     }),
     uid: instance.uid,
     translateItem,
+    el: carouselItemRef.value as Node,
```

Comment:
Maybe we can get `el` from `instance`.

```suggestion
    el: instance.vnode.el!,
```

Reply:
But we can't get help from code completion, because of `RendererNode` type is less specific than `HTMLDivElement` type. I would like using `Template Refs` .  Or how about using type assertions?
```
el:  instance.vnode.el! as HTMLDivElement
```